src/openforms/validations/api/views.py

```diff
@@ -29,7 +41,19 @@ class ValidatorsListView(ListMixin, APIView):
     serializer_class = ValidationPluginSerializer
 
     def get_objects(self):
-        return list(register.iter_enabled_plugins())
+        plugins = register.iter_enabled_plugins()
+        param_component = self.request.query_params.get("component", None)
+
+        if param_component is None:
+            return plugins
+
+        serializer = ValidationRequestParamSerializer(data=self.request.query_params)
+        if not serializer.is_valid():
+            return []
+
+        # import pdb;pdb.set_trace()
```

Comment:
This should use the filter mechanism, for a reference check DRFs base views: https://github.com/encode/django-rest-framework/blob/master/rest_framework/generics.py#L144

This would be useful to add to the `ListMixin` so that it acts more similar to the DRF `ListView`.